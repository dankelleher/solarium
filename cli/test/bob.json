[246,19,125,231,122,178,140,252,192,49,222,153,6,32,88,174,67,220,229,77,18,32,189,118,151,77,232,58,76,45,156,70,218,204,30,95,103,45,85,3,82,50,192,109,110,167,80,156,176,94,238,22,159,74,70,196,126,124,30,113,248,162,201,214]