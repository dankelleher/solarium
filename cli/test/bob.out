Generating a new keypair

Wrote new keypair to bob.json
======================================================================
pubkey: Fj6QJEa5xK95sxnXMNo4w8fzbVfdB6vdxEUp2JjCcPt9
======================================================================
Save this seed phrase and your BIP39 passphrase to recover your new keypair:
humor soup extend calm need citizen illness name knife swim cross gaze
======================================================================
